opentelemetry-instrumentation/README.rst

```diff
@@ -51,40 +52,47 @@ opentelemetry-instrument
 
 The instrument command will try to automatically detect packages used by your python program
 and when possible, apply automatic tracing instrumentation on them. This means your program
-will get automatic distributed tracing for free without having to make any code changes
-at all. This will also configure a global tracer and tracing exporter without you having to
-make any code changes. By default, the instrument command will use the OTLP exporter but
-this can be overridden when needed.
+will get automatic distributed tracing without having to make any code changes. This will
+also configure a global tracer and tracing exporter as well as a meter and meter exporter.
+By default, the instrument command will use the OTLP exporter but this can be overridden.
 
-The command supports the following configuration options as CLI arguments and environment vars:
+The command supports the following configuration options as CLI arguments and environment
+variables:
 
 
 * ``--traces_exporter`` or ``OTEL_TRACES_EXPORTER``
 * ``--metrics_exporter`` or ``OTEL_METRICS_EXPORTER``
 * ``--distro`` or ``OTEL_PYTHON_DISTRO``
 * ``--configurator`` or ``OTEL_PYTHON_CONFIGURATOR``
 
-Used to specify which trace exporter to use. Can be set to one or more of the well-known exporter
-names (see below).
+The exporter options define what exporter destination to use and can be set to one or more
+exporter names (see below). You can pass multiple values to configure multiple exporters
+(e.g., ``zipkin_json,otlp``).
 
     - Defaults to `otlp`.
     - Can be set to `none` to disable automatic tracer initialization.
+    - Can be set to 'console` to display JSON results locally.
 
-You can pass multiple values to configure multiple exporters e.g, ``zipkin,prometheus``
-
-Well known trace exporter names:
+Trace exporter names:
 
     - jaeger_proto
     - jaeger_thrift
     - opencensus
+    - otlp
+    - otlp_proto_grpc (`deprecated`)
+    - otlp_proto_http (`deprecated`)
     - zipkin_json
     - zipkin_proto
+
+Metric exporter names:
+
     - otlp
     - otlp_proto_grpc (`deprecated`)
-    - otlp_proto_http (`deprecated`)
+    - prometheus
 
 Note: The default transport protocol for ``otlp`` is gRPC.
-HTTP is currently supported for traces only, and should be set using ``OTEL_EXPORTER_OTLP_TRACES_PROTOCOL=http/protobuf``
+HTTP is currently supported for traces only, and can be set using
```

Comment:
This is no longer correct. We support HTTP for all signals https://github.com/open-telemetry/opentelemetry-python/tree/main/exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http